TABTestKit/Classes/BDD/Step.swift

```diff
@@ -7,12 +7,12 @@
 //  Created by Kane Cheshire on 25/07/2018.
 //
 
-import Foundation
+import XCTest
 
-public typealias Given = Step
-public typealias When = Step
-public typealias Then = Step
-public typealias And = Step
+open class Given: Step {}
```

Comment:
Wondering if instead of classes we could make Step a protocol instead, and then have Given/When/Then/And conform to Step? That way you'd still have lightweight structs but you'd be able to tell the difference between Given/When etc

Reply:
Great addition, the only thing left was storing the current step, so I decided to move that to TABTestCase, so now you can access it like this: `TABTestCase.current?.step`

Reply:
Ahh yes of course. What about just TABTestCase.currentStep? It can be a static property on TABTestCase rather than an instance property on TABTestCase, if that makes sense?

Reply:
fine with me :) Processed feedback